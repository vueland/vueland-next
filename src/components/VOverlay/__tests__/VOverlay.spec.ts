import { mount, VueWrapper } from '@vue/test-utils'
import { VOverlay } from '../index'

describe('VOverlay', () => {
  let mountFunction: (options?: any) => VueWrapper<any>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VOverlay, { ...options })
  })

  it('should mount component and match snapshot', () => {
    const cmp = mountFunction()

    expect(cmp.html()).toMatchSnapshot()
  })
})
