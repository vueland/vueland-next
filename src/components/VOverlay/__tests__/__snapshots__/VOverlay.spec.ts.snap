// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VOverlay should mount component and match snapshot 1`] = `""`;
